### DISCUSSION TITLE
How can I use this theme?

### DISCUSSION THREAD

**Discussion by HIDE-r** [chrisgrieser/shimmering-focus#152] [Q&A] [answered]
I followed the document,  but I can not see any change? I watch the theme's css file, all there just is code comment.
> You can find Shimmering Focus in Obsidian’s community themes browser under
Obsidian Settings ➞ Appearance: Themes ➞ Manage.

I try to install `style setting` plugin and toggle some option.  But I have not found any changes too. So I have some confusion, I hope anyone to give me help~

**Comment by chrisgrieser** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/73286100/166142259-03ba7158-5c6a-4650-9904-60145ee3b895.png)

After that, search for "Shimmering Focus".

---

However, be aware that if you are new to Obsidian, the theme probably isn't for you, since it does hide a lot of UI elements (assuming you are already so familiar with Obsidian that you know the keyboard shortcuts for them)

**Reply by HIDE-r** [reply]
After I installed the theme, it look like
![image](https://user-images.githubusercontent.com/51672711/166150871-20b40622-d1ee-48c6-803a-e15341ddf22a.png)
![image](https://user-images.githubusercontent.com/51672711/166150834-8aff07d5-385b-4665-a6ee-93fdb3917af4.png)

This looks no different from the default theme. Is this the result of expectation?

**Reply by HIDE-r** [reply]
I just changed base theme to 'light' and Updated the theme, It working now! Is it because of something is updated in up-stream? or my action had something special significance？

**Reply by chrisgrieser** [reply]
Ah no fault on your part, there was an issue with the built automation (#154). 

Issue is fixed now :)
